lib/include/modelSpec.h

```diff
@@ -157,8 +166,14 @@ class NNmodel
     //! Find a neuron group by name
     NeuronGroup *findNeuronGroup(const std::string &name);
 
-    NeuronGroup *addNeuronPopulation(const string&, unsigned int, unsigned int, const double *, const double *); //!< Method for adding a neuron population to a neuronal network model, using C++ string for the name of the population
-    NeuronGroup *addNeuronPopulation(const string&, unsigned int, unsigned int, const vector<double>&, const vector<double>&); //!< Method for adding a neuron population to a neuronal network model, using C++ string for the name of the population
+    //! Find a MPI neuron group by name
+    const NeuronGroup *findMPINeuronGroup(const std::string &name) const;
```

Comment:
See https://github.com/genn-team/genn/pull/158/commits/ea55f4c146977d32ed45b3d26ea404c7949462e4

Reply:
In fact, it is not. map insert in C++ just duplicate elements. So the same name element in m_LocalNeuronGroups is not the same one in m_NeuronGroups. As you said below, I think I should change the structure so that m_NeuronGroups contains pointers but not the duplicated elements.

Reply:
Yes - there definitely should not be duplicate neuron groups - especially as, using the non-const versions of these methods - you can modify either one independently!